src/apiProxy/apiConfigCreators.js

```diff
@@ -4,6 +4,7 @@
  * API
  *
  * @module apiConfigCreators
+ * @deprecated
```

Comment:
The entire module is deprecated, but I wanted to leave it around until our current apps (mup-web, admin-web, starter) have refactored their queries to not need it. Since a few people are using the `npm link` method of making updates to the platform alongside app changes, I didn't want them to get in a confusing state where this had disappeared from `master` before they were ready - I'll do followups in the next sprint to change over those apps